pandas/core/groupby/groupby.py

```diff
@@ -721,6 +722,10 @@ def f(g):
                     with np.errstate(all="ignore"):
                         return func(g, *args, **kwargs)
 
+            elif hasattr(nanops, "nan" + func):
+                # TODO: should we wrap this in to e.g. _is_builtin_func?
```

Comment:
That would seem logical; follow ups to figure out what is actually needed here are certainly welcome